shell/platform/android/test/io/flutter/embedding/engine/FlutterEngineGroupComponentTest.java

```diff
@@ -127,4 +139,21 @@ public void canSpawnMoreEngines() {
             RuntimeEnvironment.application, mock(DartEntrypoint.class));
     assertEquals(2, engineGroupUnderTest.activeEngines.size());
   }
+
+  @Test
+  public void canCreateAndRunCustomEntrypoints() {
+    FlutterEngine firstEngine =
+        engineGroupUnderTest.createAndRunEngine(
+            RuntimeEnvironment.application,
+            new DartEntrypoint(
+                FlutterInjector.instance().flutterLoader().findAppBundlePath(),
```

Comment:
This is unfortunate, this is impossible to find for people that want to make a DartEntrypoint.  Maybe we could add a const for it in the DartEntrypoint class?   `DartEntrypoint.DEFAULT_APP_BUNDLE_PATH`?  I couldn't find an example of anyone using the new API on github.  It looks like a series of 2 changes have been deprecated and replaced for this.

Reply:
Totally agree. This API makes no sense lol. But that's a "new feature" and I want to cherry pick this. Left https://github.com/flutter/flutter/issues/74828. Surprised no one complained yet :)